apps/modernization-ui/src/apps/search/layout/result/SearchResults.tsx

```diff
@@ -17,17 +17,44 @@ const SearchResults = ({ children, onRemoveTerm }: Props) => {
 
     const { view, terms } = useSearchResultDisplay();
 
+    const headerRef = useRef<HTMLDivElement>(null);
+    const paginationRef = useRef<HTMLDivElement>(null);
+    const [contentHeight, setContentHeight] = useState<string>('auto');
+
+    const updateContentHeight = () => {
+        requestAnimationFrame(() => {
+            const headerHeight = headerRef.current?.offsetHeight || 0;
+            const paginationHeight = paginationRef.current?.offsetHeight || 0;
+            setContentHeight(`calc(100% - ${headerHeight}px - ${paginationHeight}px)`);
+        });
+    };
+
+    useEffect(() => {
+        updateContentHeight();
+        window.addEventListener('resize', updateContentHeight);
+
+        return () => window.removeEventListener('resize', updateContentHeight);
+    }, []);
+
+    useEffect(() => {
+        updateContentHeight();
+    }, [terms]);
```

Comment:
question:  is this doing anything?  It seems like it's being updated by the resize listener